src/app/Events/EventTypes.tsx

```diff
@@ -292,7 +292,7 @@ export const EventTypes: React.FC<EventTypesProps> = (_) => {
         </Toolbar>
         {typeRowPairs.length ? (
           // TODO replace table with data list so collapsed event options can be custom formatted
-          <TableComposable aria-label="Event Types Table" variant={TableVariant.compact}>
+          <TableComposable aria-label="Event Types table" variant={TableVariant.compact}>
```

Comment:
same here about `event types`, I guess in this case since it's a separate string as a label it should be capitalized since it's a sentence (fragment): `Event types table`?